js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "braintrust",
-  "version": "0.0.160",
+  "version": "0.0.161",
```

Comment:
we update this via `version.py` when we bump the versions themselves. no need to here

Reply:
gotcha. reverted.